pwa/src/templates/categoriesDetailTemplate/CategoriesDetailTemplate.tsx

```diff
@@ -54,32 +54,54 @@ export const CategoriesDetailTemplate: React.FC<CategoriesDetailTemplateProps> =
           <LeadParagraph className={styles.description}>{portfolio.longDescription}</LeadParagraph>
         </div>
       )}
-      {component && (
+      {components && (
         <div className={styles.solutions}>
-          <div className={styles.solutionsHeader}>
-            <span className={maxItems && styles.maxNumber}>
-              <BadgeCounter number={badgeLabel}>
-                <Heading3 className={styles.title}>{t("Solutions")}</Heading3>
-              </BadgeCounter>
-            </span>
+          <div className={styles.viewButtons}>
+            <Button
+              variant={viewType === "cards" ? "primary-action" : "secondary-action"}
+              onClick={() => setViewType("cards")}
+            >
+              {t("Cards")}
+            </Button>
+            <Button
+              variant={viewType === "layers" ? "primary-action" : "secondary-action"}
+              onClick={() => setViewType("layers")}
+            >
+              {t("Layers")}
+            </Button>
```

Comment:
You are missing the icons for the buttons. `faGripVertical` for cards and `faLayerGroup` for the layers.
You can check the componentsTemplate.tsx for reference.

Reply:
Dont forget to add the buttonIcon css.